packages/ui/src/components/va-counter/VaCounter.vue

```diff
@@ -0,0 +1,391 @@
+<template>
+  <VaInputWrapper
+    class="va-counter"
+    v-bind="fieldListeners"
+    :class="computedClass"
+    :style="computedStyle"
+    :color="$props.color"
+    :readonly="$props.readonly"
+    :disabled="$props.disabled"
+    :messages="$props.messages"
```

Comment:
How about to use `extractComponentProps` function? Also, how about rules? Do we respect validation in this component?

Reply:
1. I decided to use lodash `pick` function to choose props for `VaInputWrapper`.
2. We discussed and decided that this component can be without validation. For validation, it has props `min`, `max` and `step`